### DISCUSSION TITLE
ActiveCampaign

### DISCUSSION THREAD

**Discussion by wedot** [solspace/craft-freeform#97] [Feature Requests]
Can not connect FreeForm to the ActiveCampaign contact list. I'd like to use Free Form to add Subscribers to the ActiveCampaign contact list. It would be very nice to add this option to the Plugin.

**Comment by kjmartens** [comment]
Just to clarify, you're wanting to add submission data to a Contact only (not a mailing list)?

If this is the case, you could try out the CRM integration instead. I'm not 100% sure, but it might just let you map submissions to Contacts only, but it might also not work if it requires mapping to a Deal (haven't tested this use-case yet):
https://docs.solspace.com/craft/freeform/v3/api-integrations/crm/activecampaign.html

**Comment by wedot** [comment]
Right, this is that I try to do. I will test it with the CRM integration. I think it will create a Deal in the CRM. 

I don't know if it is possible to just add a Contact without to subscribt it to a list. Maybe I need to make a list for all new contacts.

**Comment by kjmartens** [comment]
@wedot The latter approach might be your way around it. Include a mailing list (whether you use it or not) and subscribe them to that in order to have the Mailing List version of ActiveCampaign work (until we can revise it). :)

**Comment by kjmartens** [comment]
The form builder and integrations have been overhauled in Freeform 5 for Craft 4 (now in beta). You can now choose to map to some or several endpoints in each integration. 🙂
